types/mousetrap/index.d.ts

```diff
@@ -18,19 +18,20 @@ declare namespace Mousetrap {
         stopCallback: (e: ExtendedKeyboardEvent, element: Element, combo: string) => boolean;
         bind(
             keys: string | string[],
-            callback: (e: ExtendedKeyboardEvent, combo: string) => any,
+            callback: (e: ExtendedKeyboardEvent, combo: string) => void,
             action?: string,
         ): MousetrapInstance;
         unbind(keys: string | string[], action?: string): MousetrapInstance;
         trigger(keys: string, action?: string): MousetrapInstance;
         reset(): MousetrapInstance;
+        init(): void;
```

Comment:
```suggestion
```
isn't that one internal? Undocumented?

Reply:
It is not listed on document site. I'll remove it.